core/src/main/java/io/kestra/core/schedulers/AbstractScheduler.java

```diff
@@ -332,18 +336,9 @@ private void handle() {
                     .conditionContext(flowWithTriggers.getConditionContext())
                     .triggerContext(flowWithTriggers.TriggerContext.toBuilder().date(now()).stopAfter(flowWithTriggers.getAbstractTrigger().getStopAfter()).build())
                     .build())
-                .filter(f -> f.getTriggerContext().getEvaluateRunningDate() == null && !f.getTriggerContext().getDisabled())
```

Comment:
Why do you want to evaluate disabled triggers?

Reply:
You did the filtering of disabled triggers earlier :
![image](https://github.com/kestra-io/kestra/assets/37600690/84be2b1f-40a7-48e4-a607-849135b42a6e)